packages/scalars/src/typeDefs.ts

```diff
@@ -0,0 +1,3 @@
+export const EmailAddressTypeDefinition = 'scalar EmailAddress';
```

Comment:
If the type definition is exported by each scalar then I don't think this file is needed.